src/containers/app/index.scss

```diff
@@ -20,5 +20,5 @@
 }
 
 .appContent {
-  margin: 50px 0; // for fixed navbar
+  top: 75px; // for fixed navbar
```

Comment:
I am not too sure about this change. I think you should revert it. I bet it was something you had to add before you moved the welcome page out of the App wrapper page.